src/js/services/settings.js

```diff
@@ -0,0 +1,27 @@
+import { get } from 'underscore';
+import Radio from 'backbone.radio';
+
+import App from 'js/base/app';
+
+export default App.extend({
+  channelName: 'settings',
+  radioRequests: {
+    'get': 'getSetting',
+  },
+  initialize({ settings }) {
+    this.settings = settings;
+  },
+  getSetting(settingName) {
```

Comment:
No longer need the `if (!this.isRunning()) return;` check?

Reply:
correct because it's now started after the bootstrap is started it's init'd only when needed and it no long is `start`ed.  It _might_ be possible to get a warning if this setting gets requested prior to it starting, but I _think_ that was only oddly possible in testing, and generally the warning is more useful than the `return;` I would expect.